### ISSUE TITLE
Suggestion to Add Custom Multi-turn Dialogues for Few-shot Learning in Chatbot and Agent Apps

### ISSUE THREAD

**Issue by neotize** [langgenius/dify#8707]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I suggest adding a feature similar to the 'Add Message' option in the LLM node of workflows to Chatbot and Agent type apps (customizable multi-turn USER and ASSISTANT dialogues) to achieve a few-shot effect.
![QQ截图20240924105629](https://github.com/user-attachments/assets/7eccac36-c8cb-4bb1-9959-c4039691f980)

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by crazywoola** [comment]
Can you tell us why you need this separate component, too fine-grained components will lead to increased maintenance costs, so are there any special use cases

**Comment by neotize** [comment]
> Can you tell us why you need this separate component, too fine-grained components will lead to increased maintenance costs, so are there any special use cases

In our company's practice, we found that most models can be effectively guided in terms of speaking style through a few simple USER and ASSISTANT dialogue examples. By using the 'Add Message' feature in the LLM node of Dify workflows, we achieved great results. However, this feature is not supported in Chatbot or Agent type apps. Therefore, we hope that the Agent node can support the 'Add Message' feature, or as mentioned in another issue (https://github.com/langgenius/dify/issues/7470), support the Agent node in workflows to achieve the desired effect.
